packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -964,7 +964,7 @@ export const toStyles = (
 		}
 
 		const classes = getPresetsClasses( selector, presets );
-		if ( ! isEmpty( classes ) ) {
+		if ( classes.length > 0 ) {
```

Comment:
`classes` will always be a string: 

https://github.com/WordPress/gutenberg/blob/4a6972b681196432663b9f4c5ce53efb8ba80b49/packages/block-editor/src/components/global-styles/use-global-styles-output.js#L136